code/modules/antagonists/heretic/magic/flesh_surgery.dm

```diff
@@ -91,17 +91,15 @@
 		return FALSE
 
 	var/organ_hp_to_heal = to_heal.maxHealth * organ_percent_healing
-	if(to_heal.damage < organ_hp_to_heal)
-		to_heal.set_organ_damage(organ_hp_to_heal)
-		to_heal.balloon_alert(caster, "organ healed")
-		playsound(to_heal, 'sound/magic/staff_healing.ogg', 30)
-		new /obj/effect/temp_visual/cult/sparks(get_turf(to_heal))
```

Comment:
IMO, the behavior of it not playing sfx / vfx if no damage was healed should be maintained

Reply:
Alright, I can keep that, but it will be a pretty rare occurence (organs usually decay continuously, so there's almost always damage to heal)